### PR TITLE
minimal changes to upgrade to angular.js 1.3.8 the prophetic narwhal

### PR DESCRIPTION
**Pull request by pedantix**
1. upgraded the angular libraries from 1.2.X to release edge which is 1.3.8
2. tested the example project both with jasmine/test'em and protractor
3. there was one incompatibility in the router's html5 settings there were two solutions however the route seemed to change the code the lease by not setting a base URL.

### CODE REVIEW

**Comment by searls** [comment]
@davemo does this look good?

File: app/js/router.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 angular.module("app").config(function($routeProvider, $locationProvider) {
 
-  $locationProvider.html5Mode(true);
+  $locationProvider.html5Mode({enabled:true, requireBase: false});
```
Comment (davemo) [inline app/js/router.js]: I think it would make more sense to add the required `<base href="/">` to `pages/index.us` instead of switching this off. :)

**Comment by davemo** [comment]
If you can make the change to `pages/index.us` it looks good @pedantix :)

**Comment by pedantix** [comment]
@davemo I am on it

**Comment by davemo** [comment]
Looks good, thanks @pedantix  :)

**Comment by pedantix** [comment]
Thanks @davemo I am looking forward to learning angular on lineman

**Pull request merged**